examples/notebooks/nvidia.ipynb

```diff
@@ -355,4 +362,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 4
-}
+}
```

Comment:
Don't know why it's not letting me comment on the exact line, but some notebooks have a
```
    "nbmake": {
     "post_cell_execute": [
      "# Deletes collection for test suite to allow each test to run with a fresh collection",
      "vstore.delete_collection()"
     ]
}
```

which can now be removed thankfully.

But doesn't matter also, we can remove those whenever.